packages/ketcher-core/src/domain/entities/atom.ts

```diff
@@ -46,6 +46,13 @@ export interface AtomAttributes {
   substitutionCount?: number;
   ringBondCount?: number;
   explicitValence?: number;
+  /**
+   *  Values can be `0 | 1 | 2 | 3 | null`.
+   * `1` - has a `primary` R-Group Attachment Point
+   * `2` - has a `secondary` R-Group Attachment Point
+   * `3` - has a `primary` and a `secondary` R-Group Attachment Points
+   * `null` and `0` both mean the atom has no R-Group Attachment Points
+   */
   attpnt?: any;
```

Comment:
Curious, can we also remove the 'any' type and make it like this:

```typescript
attpnt?: 0 | 1 | 2 | 3 | null;
```

Reply:
Yes this is what I wanted, but considering my current task #2742 is using a lot `attpnt`. I'm afraid it would take efforts to resolve conflicts so I created a refactoring request https://github.com/epam/ketcher/issues/2962